media-libs/opencv/opencv-4.4.0.ebuild

```diff
@@ -56,6 +57,7 @@ RDEPEND="
 		media-libs/freetype:2[${MULTILIB_USEDEP}]
 		media-libs/harfbuzz:=[${MULTILIB_USEDEP}]
 	)
+	contribovis? ( dev-games/ogre:0/1.12 )
```

Comment:
Ogre must be built with [deprecated] USE flag, as OVIS uses nodeless lights that won't compile without having that flag on OGRE.
So it should be:
contribovis? ( dev-games/ogre:0/1.12 )[deprecated]

Reply:
Last time I checked, ogre-1.12.8 doesn't need to enable deprecated USE flag for ovis module to build with. Does it have effects on runtime functionalities?

Reply:
No, OVIS uses nodeless lighting (At leastopencv-4.1.2 did) and that is no longer available without that USE flag.
If opencv-4.4.0 has switched to the correct lighting (attached to nodes), then you are right. Ogre does not need to be built with the deprecated USE flag, then.